Apps/WebClient/src/WebClient.csproj

```diff
@@ -52,7 +52,6 @@
   </ItemGroup>
   <ItemGroup>
     <Folder Include="Views\DumpHeaders\" />
-    <Folder Include="Views\Robots\" />
```

Comment:
any chances that we have removed this by accident?

Reply:
No, I mentioned it in the description of the PR:
_I removed an obsolete folder in the WebClient codebase related to Robots._

Reply:
nice, my bad for not reading your comments on that :-)